doc/langdef.md

```diff
@@ -2152,14 +2152,30 @@ See [cel-go/issues/9](https://github.com/google/cel-go/issues/9).
     </td>
   </tr>
   <tr>
-    <th rowspan="4">
+    <th rowspan="8">
       size
     </th>
+    <td>
+      string.() -> int
+    </td>
+    <td>
+      number of unicode codepoints in the string
+    </td>
+  </tr>
+  <tr>
     <td>
       (string) -> int
```

Comment:
Mark the function-style overload as "deprecated", here and below.  (Boy, I wish we had adapted [Uniform Function Call Syntax](https://en.wikipedia.org/wiki/Uniform_Function_Call_Syntax) and we wouldn't have had to worry about any of this.)